### DISCUSSION TITLE
[iOS] Any plans to support storing the db in shared containers?

### DISCUSSION THREAD

**Discussion by gregzo** [capacitor-community/sqlite#469] [Ideas]
On iOS, it is quite customary to store databases in special containers called 'Shared Containers' which can be accessed by multiple apps belonging to the same 'App Group'. This is especially useful when implementing extensions (Share, App Intents...) since the system doesn't wake up the main app but it's extension, a separate process which cannot access the main app's sandbox.

Any feedback welcome. There are synchronisation issues to be mindful of in order to prevent concurrent writes so it isn't a completely trivial feature.

**Comment by jepiqueau** [comment]
@gregzo thanks for your comments. There is no plan to do this. The plugin offers to day one read/ write and one read simultaneous connection to a database.
